WORKSPACE

```diff
@@ -70,13 +70,13 @@ rust_register_toolchains(
         "x86_64-pc-windows-msvc",
         "x86_64-unknown-linux-gnu",
     ],
-    rust_analyzer_version = rust_common.default_version,
-    versions = ["1.76.0"],
+    rust_analyzer_version = "1.79.0",
+    versions = ["1.79.0"],
 )
 
 rust_analyzer_toolchain_tools_repository(
     name = "rust_analyzer_toolchain_tools",
-    version = rust_common.default_version
+    version = "1.79.0"
```

Comment:
Note: this version isn't even default in the newer versions of rules-rust, so I set it manually here